module1/homework/removeVowels/vowel.cpp

```diff
@@ -0,0 +1,33 @@
+#include "vowel.hpp"
+
+std::vector<char> vowels = {'a', 'e', 'i', 'o', 'u', 'y', 'A', 'E', 'I', 'O', 'U', 'Y'};
+
+bool isVowel(char &character)
+{
+    if (std::find(vowels.begin(), vowels.end(), character) != vowels.end())
+    {
+        return true;
+    }
+    else
+    {
+        return false;
+    }
+}
+
+void removeVowels(std::vector<std::string> &texts)
+{
+    for (auto &text : texts)
+    {
+        std::string textWithoutVowels = "";
+
+        for (auto character : text)
+        {
+            if (!isVowel(character))
+            {
+                textWithoutVowels += character;
+            }
+        }
+
+        text = textWithoutVowels;
+    }
```

Comment:
It is possible to use 3 or 4 lines instead. As we are practicing STL tou can try using erase and remove_if (for example)

Reply:
+1

Reply:
done ;)